bids/config.py

```diff
@@ -71,7 +70,7 @@ def from_file(filenames, error_on_missing=True):
     if isinstance(filenames, str):
         filenames = [filenames]
     for f in filenames:
-        if exists(f):
+        if Path(f).exists():
             with open(f, 'r', encoding='utf-8') as fobj:
                 settings = json.load(fobj)
```

Comment:
```suggestion
            settings = json.load(Path(f).read_text())
```

Reply:
That is certainly shorter. I would keep the encoding though.

```suggestion
            settings = json.load(Path(f).read_text(encoding='uft-8'))
```

Also, that would be a second conversion to `Path` so I would also do `f = Path(f)` at the start of the loop.